Abstract Syntax and Semantics Document/index.html

```diff
@@ -398,9 +398,89 @@
+      </p>
+      
+      <aside class="example" title="Merge of RDF Streams that Do Not Share Blank Nodes">
+        <pre><code class="highlight">
+        _:1 {:axel :isIn :RedRoom. :darko :isIn :RedRoom} {_:1, prov:generatedAtTime, t1}
+        _:2 {:axel :isIn :BlueRoom. }                     {_:2, prov:generatedAtTime, t2}
+        _:3 {:minh :isIn :RedRoom. }                      {_:3, prov:generatedAtTime, t3}
+        </code></pre>
+        is a merge of the following two streams:
+        <pre><code class="highlight">
+        _:1 {:axel :isIn :RedRoom. :darko :isIn :RedRoom} {_:1, prov:generatedAtTime, t1}
+        _:2 {:minh :isIn :RedRoom. }                      {_:2, prov:generatedAtTime, t3}
+        </code></pre>
+        <pre><code>
+        _:1 {:axel :isIn :BlueRoom. }                     {_:1, prov:generatedAtTime, t2}
+        </code></pre>
+      </aside>
+      
+      <p class="note">Given any three RDF streams, the determination of their satisfaction of the union or merge relation is a purely syntactic computation. 
+        That is, it is not necessary to consider entailments of the RDF streams or to be concerned with the values of literals.</p>
+
+      <aside class="ednote">
+        <p>It should not be hard to show that each relation defines a (deterministic) binary partial operation on the equivalence classes of isomorphic RDF streams.
+      However, note that this operation is not purely syntactic.
+      The construction of a union of two streams, <code>S1</code> and <code>S2</code>, can be accomplished, theoretically, by the following steps:</p>
+      <ol>
+        <li>Construct the set of timestamped graphs <code>R3</code> that is the set union of elements of <code>S1</code> and <code>S2</code>. </li>
+        <li>Construct an RDF stream <code>S3</code> from the set <code>R3</code> by placing this set into a sequential form that satisfies the constraint regarding relative order
+          of timestamped graphs, if possible.</li>
+      </ol>
+        <p>
+        
+The second step may require consideration of the values of timestamps or entailments regarding temporal order in order to <em>sequentialize</em> the set of timestamp graphs.
+          The construction of a merge is accomplished by performing stream copying to a common surface before union:
+        </p>
+        <ol>
+          <li>Copy both <code>S1</code> and <code>S2</code> to a new RDF surface by renaming blank nodes that occur in both streams to fresh blank nodes (new to both <code>S1</code> and <code>S2</code>),
+            producing new streams <code>S1'</code> and <code>S2'</code>. </li>
+          <li>Form the union of streams <code>S1'</code> and <code>S2'</code>.</li>
+        </ol>
+      </aside>
+      <p class="ednote">
+        The continuous operation of union or merge of two semi-infinite streams may not be computable, even when the operation is defined.
```

Comment:
here we deal with practical-implementation details. not sure if necessary to discuss this at this point